src/algokit/cli/localnet.py

```diff
@@ -57,14 +58,27 @@ def localnet_group() -> None:
 
 
 @localnet_group.command("start", short_help="Start the AlgoKit LocalNet.")
-def start_localnet() -> None:
-    sandbox = ComposeSandbox()
+@click.option(
+    "name",
+    "--name",
+    default="sandbox",
+    help="a name for your localnet",
+)
+def start_localnet(name: str) -> None:
+    sandbox = ComposeSandbox(dir_name=f"sandbox_{name}" if name != "sandbox" else name)
+    sandbox.save_current_directory()
     compose_file_status = sandbox.compose_file_status()
     sandbox.check_docker_compose_for_new_image_versions()
```

Comment:
Do we want to run this when using a named localnet? The message printed will tell the user to use `algokit localnet reset --update`, which if they run that we'll prompt to confirm environment override.

Additionally this will always check against the `latest` tag, which in the case of a named localnet, the user may lock the image in the compose file to some other tag.

I think a better option is to notify the user on start of a named environment (alongside print the config path), that we won't be checking for updates in this localnet.